packages/dcos-integration-test/extra/test_remove_dockercfg_hook.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+import pytest
+
+
+def test_remove_dockercfg_hook(dcos_api_session):
+    """Test that the remove .dockercfg hook is working properly.
+
+    If the hook is enabled, the test expects that the .dockercfg file
+    (downloaded as a uri via the Mesos fetcher) is removed from the task's sandbox.
+
+    """
+
+    # Skip the test if the hook is disabled
+    if not dcos_api_session.dockercfg_hook_enabled:
```

Comment:
This is not the way to skip this test. What you should do is decorate the test with something like:
```
from test_helpers import expanded_config

@pytest.skipif(expanded_config['dcos_remove_dockercfg_enable'] == 'true')
```